tests/v2/validation/provisioning/hosted/aks/scaling/scaling_nodepools.go

```diff
@@ -0,0 +1,98 @@
+package scaling
+
+import (
+	"time"
+
+	"github.com/rancher/rancher/tests/framework/clients/rancher"
+	management "github.com/rancher/rancher/tests/framework/clients/rancher/generated/management/v3"
+	"github.com/rancher/rancher/tests/framework/extensions/cloudcredentials"
+	"github.com/rancher/rancher/tests/framework/extensions/clusters/aks"
+	np "github.com/rancher/rancher/tests/framework/extensions/clusters/aks/resources"
+	"github.com/sirupsen/logrus"
+	kwait "k8s.io/apimachinery/pkg/util/wait"
+)
+
+// ScalingAKSNodePools is a helper function that tests scaling of an AKS node pool by adding a new one and then deleting it.
+func ScalingAKSNodePools(client *rancher.Client, oldCluster *management.Cluster, displayName string, cloudCredential *cloudcredentials.CloudCredential) (*management.Cluster, error) {
```

Comment:
Markus I just realized something, I don't think this func should be in its own package, since it's a helper not its own test. My last nitpick

Reply:
Sure thing, I'll remove it and port the change to the backport too.

Reply:
Alright, made the change to both this and the 2.6 PR. Should be good now 😄 . Thanks for the catch!